frontend/src/stores/studentStore.js

```diff
@@ -0,0 +1,139 @@
+import { action, makeObservable, observable, computed } from 'mobx';
```

Comment:
Als je naar de klas gaat en je kijkt in je scherm rechts, dan zie je allemaal warnings staan in de richting van:

`[mobx] Out of bounds read: 1`

Het is heel belangrijk om hier even goed naar te kijken, want het kan zijn dat bijv. een forloop dan ergens niet goed is ingesteld